_dashboards/get-started/time-filter.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Set the time range
+parent: OpenSearch Dashboards core concepts
+nav_order: 10
+---
+
+## Set the time range
+
+You can change the time range to display dashboard data over minutes, hours, days, weeks, months, or years.
```

Comment:
Should we also mention that we can change the default time range and time refresh interval under advanced setting page?

Reply:
Revised